rfcs/supported/README.md

```diff
@@ -0,0 +1,12 @@
+# Design Documents for Supported Features
+
+Supported proposals describe functionality implemented and
+released as fully supported features of the library. A fully supported feature
+has a high-quality implementation, is regularly tested and properly documented.
+
+If the proposal impacted the public API of the library, it should usually involve
+changes in the oneDPL specification. The RFC document should in that case have a link
+to the formal wording in the specification.
```

Comment:
Please add commas:
The RFC document should**,** in that case**,** have a link
to the formal wording in the specification.